src/app/courses/courses/courses.component.ts

```diff
@@ -24,7 +24,15 @@ export class CoursesComponent implements OnInit {
     this.courses = [];
```

Comment:
Remove unused hooks or add them to `implements` section.

Reply:
Fixed